File: sdks/thunkmetrc-go/wrapper/models/processing_job_create_adjust_processing_job_request_item.go
package models

type ProcessingJobCreateAdjustProcessingJobRequestItem struct {
    AdjustmentDate string `json:"AdjustmentDate,omitempty"`
    AdjustmentNote string `json:"AdjustmentNote,omitempty"`
    AdjustmentReason string `json:"AdjustmentReason,omitempty"`
    CountUnitOfMeasureName string `json:"CountUnitOfMeasureName,omitempty"`
    Id int `json:"Id,omitempty"`
    Packages []*ProcessingJobCreateAdjustProcessingJobRequestItemPackage `json:"Packages,omitempty"`
    VolumeUnitOfMeasureName string `json:"VolumeUnitOfMeasureName,omitempty"`
    WeightUnitOfMeasureName string `json:"WeightUnitOfMeasureName,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ProcessingJobCreateAdjustProcessingJobRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ProcessingJobCreateAdjustProcessingJobRequestItem(
    @JsonProperty("AdjustmentDate")
    val adjustmentDate: String? = null,
    @JsonProperty("AdjustmentNote")
    val adjustmentNote: String? = null,
    @JsonProperty("AdjustmentReason")
    val adjustmentReason: String? = null,
    @JsonProperty("CountUnitOfMeasureName")
    val countUnitOfMeasureName: String? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Packages")
    val packages: List<Any>? = null,
    @JsonProperty("VolumeUnitOfMeasureName")
    val volumeUnitOfMeasureName: String? = null,
    @JsonProperty("WeightUnitOfMeasureName")
    val weightUnitOfMeasureName: String? = null
)
